### ISSUE TITLE
Why has Projections and checking of coordinate systems gotten so much trickier?

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/gcd#162]
So datasets that were showing up as having the same coordinate system in GCD 6, are not in GCD 7. Watch here: https://youtu.be/GgakxZD-gIA

For first 5 minutes. You'll see this problem occuring on just the Feshie dataset, and in later 15 minutes I test with CHAMP data and it works.

**Comment by philipbaileynar** [comment]
@MattReimer let's discuss what can be done to smooth this process...

**Comment by joewheaton** [comment]
*Just some moe info....*
So, in testing the FIS, I encountered this bug again and reproduced it. My approach to troubleshoot was to browse to the shapefile with the wrong projection and right-click and look at its properties, then apply a new coordinate system in ArcCatalog. While it says it works, when you return to view its properties, it could not update the coordinate systems when they were so similar. As such, I attempted @philipbaileynar suggestion of assigning a new coordinate system by using the `Project` command in Data Management of geoprocessing tools and creating a new shape file with the new projection. This worked! As such, there is an easy work around, but it would still be nice to 'smooth this process'.
